docs/user-guide/global-configurations/deployment-window.md

```diff
@@ -0,0 +1,279 @@
+
+![Figure 25b: Deployment Dialog](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/blocked-deployment-dialog.jpg)
+
+{% hint style="warning" %}
+Not just manual trigger, deployments remain blocked even if the trigger mode is automatic. In such cases, if a new container image comes into picture, the user has to manually deploy once the deployment block is lifted.
+{% endhint %}
+
+The `Deployment History` tab will also log whether a given deployment was initiated inside or outside the deployment window.
+
+![Figure 26: Deployment Log](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/deployment-log-v2.jpg)
+
+### Rollback
+
+Rolling back to an older version, by using a previously deployed image, is barred during a deployment block.
+
+![Figure 27a: Rollback Deployment](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/rollback-1.jpg)
+
+![Figure 27b: Selecting Previously Deployed Image](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/rollback-2.jpg)
+
+![Figure 27c: Rollback Dialog](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/rollback-3.jpg)
+
+
+### Deletion of CD Pipeline
+
+Go to **App Configuration** → **Workflow**. 
+
+In Devtron, deleting a CD pipeline affects the current state of the deployed application. Moreover, it might impact future deployments and you will also lose information about past deployments, i.e., Deployment History. 
+
+If you attempt to delete any CD pipeline with restricted deployment, it will show `Pipeline deletion is blocked`.
+
+![Figure 28a: Pipeline Deletion](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/pipeline-deletion-1.jpg)
+
+![Figure 28b: Pipeline Deletion Dialog](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/pipeline-deletion-2.jpg)
+
+---
+
+## Impact on Application Groups
+
+Just like application, [application groups](../application-groups.md) are also subjected to deployment windows.
+
+![Figure 29: Deployment Window in Application Group](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/app-group-blackout.jpg)
+
+Let's say you have 10 applications in your application group, and a blackout window is ongoing for 3 of them. In such a case, if you deploy your application group, those 3 applications will not get deployed. Therefore, you might experience a partial success along with an option to retry the failed deployments.
+
+![Figure 30: Partial Deployment of Application Group](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/ag-deploy.jpg)
```

Comment:
Can we pls change this screenshot. The messages column contains messages in Spanish. I believe this screenshot has been taken from figma.

Reply:
Done. Added an actual gif this time.